apple/swift-docc-render
# Update filter icons

Bug/issue #, if applicable: 91576596

## Summary

This simplifies the visual design for the following icons used in the filter:
* `ClearRoundedIcon`
* `FilterIcon`

Example:
<img width="457" alt="Screen Shot 2022-04-11 at 2 52 42 PM" src="https://user-images.githubusercontent.com/212918/162842657-fc330e52-28ca-482d-87b9-46ebc89cf148.png">

## Testing

Steps:
1. View a page with the navigator and enter some text in the filter
2. Verify that the new icons are displayed

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] ~~Added tests~~, icon change only
